docs/schema/identifiers.md

```diff
@@ -6,105 +6,107 @@ Consistent identifiers are essential to help join up open contracting data.
 * Organization identifiers are important to know who is involved in each contract;
 * Release, award and contract identifiers are important to help cross-reference information.
 
-## Types of identifiers
-
-In OCDS there are two kinds of identifiers: globally unique and local.
-
-### Globally unique identifiers
-
-Across the whole universe of OCDS publishers these identifiers refer to one specific process or organization.
-
-We create globally unique process identifiers by adding a prefix to the internal identifiers held by publishers.
-
-```{admonition} Worked Example
-:class: hint
+## Open contracting process identifier (ocid)
 
-Two government publishers (Town A and Town B) number their contracting processes from 0 upwards.
+<img src="../../_static/svg/green_compilation.svg" width="150" align="right"/>
 
-Town A publishes information on a contracting process to build a new road. Internally they know this as contract 0005.
+OCDS defines a release's `ocid` field as:
 
-Town B publishes information on a contracting process to buy textbooks for a school. Internally they also know this as contract 0005.
+```{field-description} ../../build/current_lang/release-schema.json /properties/ocid
+```
 
-When they publish their OCDS data, each government adds a unique prefix onto their internal identifiers.
+(The definition of a record's `ocid` field replaces "release" with "record".)
 
-Now Town A's contracting process has the `ocid` of 'ocds-fh349f-0005' and Town B's contracting process has the `ocid` of 'ocds-twb234-0005'.
+Since the `ocid` field is a required field and is globally unique, it can be used to join up data that is published at different times or from different systems.
 
-There is now no chance of these getting mixed up in a system which imports data from both towns.
+An `ocid` is composed of two parts:
 
-And, if an independent civil society contract monitoring group want to publish a report about implementation of Town A's road project, or Town B's text-book procurement, they have distinct identifiers they can use in their own data to refer to these.
-```
+1. An [OCID prefix](#ocid-prefix), which identifies the series of identifiers to which the `ocid` belongs
+1. An internal identifier for the contracting (or planning) process, which is unique within the series
 
-You can read more about the OCDS approach to identify organizations below.
+It is encouraged to separate the OCID prefix and the internal identifier with a hyphen (`-`).
 
-### Local identifiers
+The `ocid` is case-sensitive; in other words, the letter case of an ocid must be consistent.
 
-Not all the identifiers in OCDS need to be globally unique. Most only need to be unique among the identifiers used for the same type of object within the same scope. For example:
+To assign an `ocid` to a contracting (or planning) process, you need to register an OCID prefix and choose an internal identifier.
```

Comment:
Add a link to the guidance on registering an OCID prefix to "register an OCID prefix", or to the section later in the page "OCID prefix"

Reply:
https://github.com/open-contracting/standard/pull/1655/commits/eda23e89bd4de753a8d977ca767dd3dcde40e591